2021-10-28 17:21:28,888 INFO    MainThread:586546 [wandb_setup.py:_flush():69] setting env: {}
2021-10-28 17:21:28,888 INFO    MainThread:586546 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-28 17:21:28,888 INFO    MainThread:586546 [wandb_init.py:_log_setup():348] Logging user logs to /home/ingambe/DeepLearningCourse/3-Deep-networks/Slides/wandb/run-20211028_172128-2plum39d/logs/debug.log
2021-10-28 17:21:28,889 INFO    MainThread:586546 [wandb_init.py:_log_setup():349] Logging internal logs to /home/ingambe/DeepLearningCourse/3-Deep-networks/Slides/wandb/run-20211028_172128-2plum39d/logs/debug-internal.log
2021-10-28 17:21:28,890 INFO    MainThread:586546 [wandb_init.py:_jupyter_setup():300] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fcee73465b0>
2021-10-28 17:21:28,891 INFO    MainThread:586546 [wandb_init.py:init():381] calling init triggers
2021-10-28 17:21:28,892 INFO    MainThread:586546 [wandb_init.py:init():386] wandb.init called with sweep_config: {}
config: {}
2021-10-28 17:21:28,892 INFO    MainThread:586546 [wandb_init.py:init():430] starting backend
2021-10-28 17:21:28,892 INFO    MainThread:586546 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-28 17:21:28,926 INFO    MainThread:586546 [backend.py:ensure_launched():135] starting backend process...
2021-10-28 17:21:28,945 INFO    MainThread:586546 [backend.py:ensure_launched():139] started backend process with pid: 586580
2021-10-28 17:21:28,948 INFO    MainThread:586546 [wandb_init.py:init():435] backend started and connected
2021-10-28 17:21:29,013 INFO    MainThread:586546 [wandb_run.py:_label_probe_notebook():819] probe notebook
2021-10-28 17:21:29,023 INFO    MainThread:586546 [wandb_run.py:_label_probe_notebook():826] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-10-28 17:21:29,024 INFO    MainThread:586546 [wandb_init.py:init():494] updated telemetry
2021-10-28 17:21:29,025 INFO    MainThread:586546 [wandb_init.py:init():517] communicating current version
2021-10-28 17:21:34,027 INFO    MainThread:586546 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-10-28 17:21:35,337 INFO    MainThread:586546 [wandb_init.py:init():557] starting run threads in backend
2021-10-28 17:21:38,433 INFO    MainThread:586546 [wandb_run.py:_console_start():1605] atexit reg
2021-10-28 17:21:38,433 INFO    MainThread:586546 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.WRAP
2021-10-28 17:21:38,434 INFO    MainThread:586546 [wandb_run.py:_redirect():1516] Wrapping output streams.
2021-10-28 17:21:38,435 INFO    MainThread:586546 [wandb_run.py:_redirect():1540] Redirects installed.
2021-10-28 17:21:38,436 INFO    MainThread:586546 [wandb_init.py:init():582] run started, returning control to user process
2021-10-28 17:21:38,440 INFO    MainThread:586546 [wandb_init.py:_pause_backend():262] pausing backend
2021-10-28 17:21:38,440 INFO    MainThread:586546 [jupyter.py:save_ipynb():301] looking for notebook: 3-Deep-networks/Slides/3-Visualize NN.ipynb
2021-10-28 17:21:38,449 INFO    MainThread:586546 [wandb_init.py:_resume_backend():271] resuming backend
2021-10-28 17:21:38,605 INFO    MainThread:586546 [wandb_init.py:_pause_backend():262] pausing backend
2021-10-28 17:21:38,606 INFO    MainThread:586546 [jupyter.py:save_ipynb():301] looking for notebook: 3-Deep-networks/Slides/3-Visualize NN.ipynb
2021-10-28 17:23:23,845 INFO    MainThread:586546 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 0
2021-10-28 17:23:23,846 INFO    MainThread:586546 [wandb_run.py:_restore():1547] restore
2021-10-28 17:23:24,084 INFO    MainThread:586546 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  total_bytes: 810
}

2021-10-28 17:23:24,597 INFO    MainThread:586546 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  total_bytes: 810
}

2021-10-28 17:23:25,461 INFO    MainThread:586546 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  total_bytes: 10314
}

2021-10-28 17:23:25,566 INFO    MainThread:586546 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  total_bytes: 10454
}

2021-10-28 17:23:25,670 INFO    MainThread:586546 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  total_bytes: 10454
}

2021-10-28 17:23:25,774 INFO    MainThread:586546 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  total_bytes: 10454
}

2021-10-28 17:23:25,878 INFO    MainThread:586546 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  total_bytes: 10454
}

2021-10-28 17:23:25,982 INFO    MainThread:586546 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  total_bytes: 10454
}

2021-10-28 17:23:26,086 INFO    MainThread:586546 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  total_bytes: 10454
}

2021-10-28 17:23:26,190 INFO    MainThread:586546 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  total_bytes: 10454
}

